impl/client/src/main/javascript/web/pentaho/visual/models/i18n/model.properties

```diff
@@ -174,6 +174,13 @@ Line.description=Line Chart
 Line.props.lineWidth.label=Line Width
 Line.props.shape.label=Bullet Style
 
+## LineStacked
+LineStacked.label=Stacked Line
+LineStacked.description=Stacked Line Chart
+
+LineStacked.props.lineWidth.label=Stacked Line Width
+LineStacked.props.shape.label=Bullet Style
```

Comment:
As suggested above, these would instead be shared and belong to `LineAbstract`. Would need to change all locales.